react/logic/masterDataError.ts

```diff
@@ -0,0 +1,112 @@
+import { ApolloError } from 'apollo-client'
+import { GraphQLError } from 'graphql'
+import { ErrorTypes, concatFormPath } from 'react-hook-form-jsonschema'
+
+export type MasterDataErrorRecord = Record<string, ErrorTypes[]>
+
+interface MasterDataError {
+  Message: string
```

Comment:
Why these attributes are capitalized?

Reply:
Just because types and interfaces are usually capitalized in TS.